src/compas/CompasImportFromApi.ts

```diff
@@ -0,0 +1,61 @@
+import {customElement, html, LitElement, TemplateResult} from "lit-element";
+
+import '@material/mwc-button';
+
+import {newOpenDocEvent, newWizardEvent} from "../foundation.js";
+
+import {createLogEvent, handleError, handleResponse, parseXml} from "../compas-services/foundation.js";
+import {dispatchEventOnOpenScd} from "./foundation.js";
+
+import '../WizardDivider.js';
+import './CompasSclTypeList.js';
+import './CompasScl.js';
+import { CompasSettings } from "./CompasSettings.js";
+import { CompasCimMappingService } from "../compas-services/CompasCimMappingService.js";
+
+@customElement('compas-import-from-api')
+export default class CompasImportFromApiElement extends LitElement {
+  private async processCimFile(name: string) {
+    const url = CompasSettings().compasSettings.importFromApiUrl;
```

Comment:
Why no just hard-code this URL? This way we also don't have to pollute the Compas Settings.
It will become a Alliander specific plugin in the future.